### ISSUE TITLE
Examples not works

### ISSUE THREAD

**Issue by azat-io** [vitebook/vitebook#74]
I don't see any stories in your examples:

<img width="668" alt="image" src="https://user-images.githubusercontent.com/5698350/162419584-5597e67b-ef8c-4242-b503-6ef7e167b759.png">

**Comment by Reneuwumuhire** [comment]
At the beginning of this week examples were working but today I have seen that the examples are showing anything.

> I don't see any stories in your examples:
> 
> <img alt="image" width="668" src="https://user-images.githubusercontent.com/5698350/162419584-5597e67b-ef8c-4242-b503-6ef7e167b759.png">

**Comment by azat-io** [comment]
@Reneuwumuhire I am talking about Vitebook official example. There is no Button story: https://stackblitz.com/github/vitebook/vitebook/tree/main/examples/react

**Comment by oneezy** [comment]
Same here.

Any idea on what's going on @mihar-22 ?

I'm particularly looking to use the [Vitebook (Sveltekit)](https://stackblitz.com/github/vitebook/vitebook/tree/main/examples/svelte-kit?file=README.md) example but nothings showing up on **StackBlitz**

**Comment by SirMishaa** [comment]
Same issue for me, examples not working at all :/

**Comment by jonathanmach** [comment]
Yep, same issue here 👍 
![image](https://user-images.githubusercontent.com/12788052/174980410-46f69174-8b30-4e8d-b926-f613aa0f53a5.png)

**Comment by matsko** [comment]
Doesn't look like we're getting any response here. The last update was back in April.

@mihar-22 are you going to fix this problem? Currently if you download vitebook and load it up then `yarn vitebook:dev` doesn't work at all. It's an empty screen and there are no console or CLI errors that explain what's broken.

**Comment by mihar-22** [comment]
Hey @matsko! Sorry about all that. Vitebook broke from a series of upstream breaking changes in Vite. This is kind of why I put a alpha software warning at the top of the README, and never publicly promoted the project outside of a few small mentions. 

Good news is I've worked hard on the new version of Vitebook which I finished about a month ago. I just need to find some time to document it. I'm sorry for any delays as I have other higher priority work on my plate right now. Hopefully within the next month or so it should be ready.
